service/history/workflow/workflow_task_state_machine.go

```diff
@@ -616,6 +634,28 @@ func (m *workflowTaskStateMachine) AddWorkflowTaskFailedEvent(
 func (m *workflowTaskStateMachine) AddWorkflowTaskTimedOutEvent(
```

Comment:
nit: maybe later. looks like this should be renamed to AddWorkflowTaskStartToCloseTimedOutEvent ?

Reply:
:+1: